packages/lesswrong/lib/alignment-forum/comments/helpers.ts

```diff
@@ -3,25 +3,23 @@ import * as _ from 'underscore';
 export const commentSuggestForAlignment = async ({ currentUser, comment, updateComment }: {
   currentUser: UsersCurrent,
   comment: CommentsList,
-  updateComment: WithUpdateFunction<CommentsCollection>,
+  updateComment: (commentId: string, data: NullablePartial<DbComment>)=>Promise<void>,
```

Comment:
nit: probably slightly cleaner to export a derived type of the `updateComment` function from where the hook is defined, i.e. `export type UpdateCommentFunction = ReturnType<typeof useUpdateComment>;`